crate_universe/src/rendering.rs

```diff
@@ -492,8 +492,6 @@ impl Renderer {
     ) -> Result<CargoBuildScript> {
         let attrs = krate.build_script_attrs.as_ref();
 
-        const COMPILE_DATA_GLOB_EXCLUDES: &[&str] = &["**/*.rs"];
```

Comment:
It looks like we're dropping a default value - is that intentional?

Reply:
Yes, but i do not remember why, at first i left it as default, i will recheck. Maybe i missed something.

Reply:
I think i found out, it was my mistake, at first i added default to both `CommonAttributes` and `BuildScriptAttributes` this was unintended for `CommonAttributes`, so it broke the build. And i missed what i only neded to add default to `BuildScriptAttributes` (as it only used here). 
Added new commit which adds proper default here.